api/clients/v2/cert_verifier.go

```diff
@@ -32,3 +35,34 @@ type ICertVerifier interface {
 	// "available".
 	GetConfirmationThreshold(ctx context.Context, referenceBlockNumber uint64) (uint8, error)
 }
+
+// IGenericCertVerifier is an interface for interacting with the updated EigenDACertVerifier contract
+// that takes a low-level bytes interface and is compatible with the EigenDACertVerifierRouter contract.
+type IGenericCertVerifier interface {
```

Comment:
Why did you create a new `IGenericCertVerifier` interface, instead of modifying the existing interface?